src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs

```diff
@@ -408,9 +408,11 @@ public async Task<ActionResult> OrganisationDetails(OrganisationViewModel model)
 
             var existingOrgs = await GetExistingOrganisations(model);
 
-            if (existingOrgs.Any())
+            if (existingOrgs.Organisations.Any())
             {
-                return await OrganisationFound(existingOrgs);
+                TempData["FoundOrganisations"] = existingOrgs;
```

Comment:
@will-hope98  I think we can test that TempData has been set also, I think by controller.TempData["FoundOrganisations"].should().be.....